### ISSUE TITLE
fix or remove extrapolation please and ty

### ISSUE THREAD

**Issue by FeFari** [haxball/haxball-issues#1046]
hello @basro and devs. how are you?

extrapolation command has killed modern haxball. we understand how it works and despite you might think it's a nice addition to allow people play in foreign countries, it is NOT.

extrapolation command seems to work based on user's hardware. some people can fetch 100-150 extrapolation and see their player move smoothly, whereas others put 10-15 and their pc literally gets stuck. 

what's the result? extrapolations players desynch their wall liftings/rockets and also become immovable 1 on 1. this creates a clear disadvantage between players of the same country.

some of us would like to see it fixed, capable of being disable as another room option or totally removed.

thanks in advance, regards.

**Comment by rbustos93** [comment]
I agree with @FeFari , extrapolation sucks

**Comment by kingjasko** [comment]
https://www.youtube.com/watch?v=jvzzZ9Q24H0

This is one of the best if not the best player in European Haxball scene playing on 200 extrapolation. I wonder if people would play with that much extrapolation and experience such a shitty gameplay if they weren't gaining anything over others.

**Comment by FeFari** [comment]
nope, they wouldnt. but thats because his hw setup cant handle 200 extra (almost nobody can). 

in our league some ppl would use 150 without issues because they have gamer PCs, video boards and 144 hz monitors. im not saying they have to downgrade their hardware, but for example U processors lose a lot of accuracy over 50 extra. they already take adv from mech kbs, imagine also taking adv from extra...
